hera_pspec/pspecdata.py

```diff
@@ -750,54 +826,61 @@ def get_MW(self, G, mode='I'):
             Window function matrix, W. (If G was passed in as a dict, a dict of
             array_like will be returned.)
         """
+
+        ### Next few lines commented out because (while elegant), the situation
+        ### here where G is a distionary is not supported by the rest of the code.
         # Recursive case, if many G's were specified at once
-        if type(G) is dict:
-            M,W = {}, {}
-            for key in G: M[key], W[key] = self.get_MW(G[key], mode=mode)
-            return M, W
+        # if type(G) is dict:
+        #     M,W = {}, {}
+        #     for key in G: M[key], W[key] = self.get_MW(G[key], mode=mode)
+        #     return M, W
 
         # Check that mode is supported
         modes = ['G^-1', 'G^-1/2', 'I', 'L^-1']
         assert(mode in modes)
 
         # Build M matrix according to specified mode
         if mode == 'G^-1':
-            M = np.linalg.pinv(G, rcond=1e-12)
-            #U,S,V = np.linalg.svd(F)
-            #M = np.einsum('ij,j,jk', V.T, 1./S, U.T)
+            raise NotImplementedError
```

Comment:
Please say what's not implemented, e.g. `NoteImplementedError("G^-1 mode not currently supported.")`

Reply:
Done!